code/bml_builder.js

```diff
@@ -0,0 +1,87 @@
+(function() {
```

Comment:
I think we should call this file `bookmarklet-builder.js`, avoiding `bml` acronym